FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/FlowcryptApiRepository.kt

```diff
@@ -86,29 +85,33 @@ class FlowcryptApiRepository : ApiRepository {
 
   override suspend fun submitPubKey(
```

Comment:
I would rather have two methods here, `submitPrimaryEmailPubKey` and `submitPubKeyWithConditionalEmailVerification`.

These long names is so that it's very descriptive what is going on when using them. By "hiding" them behind an interface where the only difference is null/nonnull id token, this gets obscured.